ee/dtr/admin/install/index.md

```diff
@@ -26,14 +26,14 @@ Before installing DTR, confirm that your infrastructure meets DTR's [system requ
 
 ## 2. Install UCP
 
-To run, DTR requires Docker Enterprise's Docker Universal Control Panel (UCP). If UCP is not yet installed, refer to [install UCP for production](/ee/ucp/admin/install/).
+To run, DTR requires Docker Enterprise's UCP. If UCP is not yet installed, refer to [install UCP for production](/ee/ucp/admin/install/).
 
 ## 3. Install DTR
 
 >**Note**
 >
 > Prior to installing DTR:
-> * When upgrading, upgrade UCP before DTR for each major version. For example if you are upgrading four major versions, upgrade one major version at a time, first UCP, then DTR, and then repeat for the remaining three versions.
+> * When upgrading, upgrade UCP before DTR for each major version. For example, if you are upgrading four major versions, upgrade one major version at a time, first UCP, then DTR, and then repeat for the remaining three versions.
```

Comment:
I know what this is saying but I believe examples are the best way to explain things.
Referencing this: https://success.docker.com/article/compatibility-matrix

You can say something like if you have UCP: 3.0 and DTR:2.5 then the flow should be
1. UCP: 3.0 -> 3.1 upgrade
2. DTR: 2.5 -> 2.6 upgrade

3. UCP: 3.1 -> 3.2 upgrade
4. DTR: 2.6 -> 2.7 upgrade

Reply:
I agree that an example helps. I would also think about combining this note with the intro to the Install UCP section. It seems like overkill to have two entries for the same thing so close to each other.

Reply:
Reading more carefully through this section, I now think that the information should be part of the Install UCP section (and not necessarily as a Note).

Reply:
So I rejigged the 2. Install UCP section to include the pertinent information here, but decided that the UCP-DTR progression clouds up the Install DTR path. Far better served in **Upgrade DTR**, with a slight mention and xref here in **Install Docker Trusted Registry**.